examples/nextjs-keystone/keystone.ts

```diff
@@ -17,5 +17,16 @@ export default withAuth(
     },
     lists,
     session,
+    storage: {
+      my_images: {
+        kind: 'local',
+        type: 'image',
+        generateUrl: path => `http://localhost:4000/images${path}`,
```

Comment:
We might have to set the URL using environment variables. Apps deployed on vercel will have their URL defined in the env variable `process.env.NEXT_PUBLIC_VERCEL_URL`.